design/schema/mappings.csv

```diff
@@ -0,0 +1,92 @@
+BCDC,BIL,DANDI,NeMO(BDBag),BossDB
+Collection.accessControl,Dataset.Rights,Dandiset.access,BDBag.DataAccess,Collection.Public
+Collection.bkpWebResource,,,,
+Collection.citation,,Dandiset.citation,BDBag.FullDataSetCitation,
+Collection.completionState,,,,
+Collection.dataArchive,,Dandiset.repository,,
+Collection.dateCreated,,Dandiset.dateCreated,,Collection.DateCreated
+Collection.dateModified,,Dandiset.dateModified,,Collection.DateModified
+Collection.description,Dataset.Abstract,Dandiset.description,BDBag.DataSetDescription,Collection.Description
+Collection.doi,,Dandiset.doi,,Collection.DOI
+Collection.externalId,,Dandiset.identifier,BDBag.NeMOIdentifier,Collection.ID
+Collection.handle,,,BDBag.ExternalIdentifier,
+Collection.license,Dataset.Rights,Dandiset.license,BDBag.DataLicense,Collection.License
+Collection.modality,Dataset.generalModality+Other,,,
+Collection.rrid,,,BDBag.DataCollectionRRID,
+Collection.shortTitle,,,,
+Collection.species,,,BDBag.StudyOrganism,Collection.Species
+Collection.specimenCount,,,,
+Collection.technique,Dataset.technique+Other,,BDBag.ProtocolType,
+Collection.title,Dataset.Title,Dandiset.name,BDBag.DataSetTitle,Collection.Name
+Collection.webResource,,Dandiset.url,BDBag.SourceDataURL,
+License.handle,,,,
+License.name,Dataset.rightsIdentifier,,,
+License.shortTitle,Dataset.Rights,,,
+License.title,Dataset.Rights,,,
+License.webResource,Dataset.rightsURI,,,
+Project.bkpWebResource,,,,
+Project.collection,,Dandiset.wasGeneratedBy[inverse],,Project.Collections
+Project.contact,,,BDBag.Contact+Information,
+Project.contributor,,Dandiset.contributor/Project.wasAssociatedWith,BDBag.DataContributors+Roles,Project/Collection.Contributor
+Project.creator,,Dandiset.contributor,BDBag.Organization/BDBag.Authors,Collection.Creator
+Project.dataArchive,,,,
+Project.dateCreated,,,,Project.DateCreated
+Project.dateModified,,,,Project.DateModified
+Project.description,,Project.description,,Project.Description
+Project.doi,,,,Project.doi
+Project.externalId,,Project.identifier,,Project.Name
+Project.fundingSource,,,BDBag.GrantSupport,Project.Funding
+Project.handle,,,,
+Project.highlightedWebResource,,,,Project.Links
+Project.informationWebResource,,,,
+Project.modality,,,,Project.ImagingModalities/Specific
```

Comment:
For bossdb, this is a possible link!
Collection.modality->project.imagingmodalities (a project could be multi-modal in bossdb)

Reply:
Added, thank you!